beefy-node/runtime/src/lib.rs

```diff
@@ -388,10 +388,6 @@ impl_runtime_apis! {
 		) -> sp_inherents::CheckInherentsResult {
 			data.check_extrinsics(&block)
 		}
-
-		fn random_seed() -> <Block as BlockT>::Hash {
```

Comment:
What Substrate PR is related to this change?

Reply:
[primitives: remove random_seed from BlockBuilder API](https://github.com/paritytech/substrate/commit/5f73ba1f67c33d7e92a66e78a3cfd1132fa5a7ad)